### PR TITLE
Add DCU as submodule

### PR DESCRIPTION
**Pull request by EduKav1813**

### CODE REVIEW

File: dasharo-compatibility/dcu.robot [hunk 1]
```diff
@@ -147,7 +147,6 @@ DCU004.001 Verify SMMSTORE changes
 *** Keywords ***
 Prepare DCU Test Environment
     Run    rm -rf dcu
```
Comment (macpijan) [inline dasharo-compatibility/dcu.robot:149]: Do we really want to remove it still, if it as added as a submodule, and not cloned later on?

Comment (macpijan) [reply dasharo-compatibility/dcu.robot:149]: Applies to the other cases as well, if makes sense

Comment (philipandag) [reply dasharo-compatibility/dcu.robot:149]: @EduKav1813 
I am working on reworking how DCU is used in OSFV and our PRs are in conflict :). I'll update the tests to use the submodule you added here: https://github.com/Dasharo/open-source-firmware-validation/pull/543 so you won't have to do it in this PR, ok?

Comment (EduKav1813) [reply dasharo-compatibility/dcu.robot:149]: @philipandag Sure, in return I can help you test this change in your PR/reword

Comment (philipandag) [reply dasharo-compatibility/dcu.robot:149]: It was quickly tested on a single test case here: https://github.com/Dasharo/open-source-firmware-validation/issues/541#issuecomment-2482804129

You can check the whole suite to make sure then

**Comment by macpijan** [comment]
Integrated in: https://github.com/Dasharo/open-source-firmware-validation/issues/541#issuecomment-2482804129

**Pull request closed without merging**